manifests/04_oauthserver_trust_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  name: system:openshift:oauth-servercert-trust
```

Comment:
I see the `system:openshift:operator:authentication` already binds to `cluster-admin`. So do we need this spacial file here? We already should have necessary permissions.

Reply:
this role is not for the authentication operator, but for everyone (see the rolebinding below)

Reply:
For me it's fine. But please let someone more experienced have a look at this piece as well.